CalculatorKata/Program.cs

```diff
@@ -7,6 +7,20 @@ public class Program
         static void Main(string[] args)
         {
         }
-       
+        public int add(String numbers)
+        {
+            if (string.IsNullOrWhiteSpace(numbers))
+                return 0;
+            string[] num = numbers.Split(',');
+            int sum = 0;
+            if (num.Length == 2)
+                sum = Convert.ToInt32(num[0]) + Convert.ToInt32(num[1]);
```

Comment:
Is there a reason for using Convert.ToInt32 instead of int.TryParse() or int.Parse()